src/extension-support/extension-manager.js

```diff
@@ -24,7 +24,8 @@ const builtinExtensions = {
     makeymakey: () => require('../extensions/scratch3_makeymakey'),
     boost: () => require('../extensions/scratch3_boost'),
     gdxfor: () => require('../extensions/scratch3_gdx_for'),
-    vizblocks: () => require('../extensions/scratch3_vizblocks')
+    vizblocks: () => require('../extensions/scratch3_vizblocks'),
+    linechart: () => require('../extensions/scratch3_linechart')
```

Comment:
Refer to https://github.com/VisualizationLiteracyPlayground/scratch-gui/pull/3#pullrequestreview-366963137.